tests/functional/pv/space_reclaim/test_rook_reclaim_namespace.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import time
+
+import pytest
+
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.resources.pod import get_csi_provisioner_pod
+from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.helpers.helpers import (
+    create_reclaim_space_job,
+    verify_log_exist_in_pods_logs,
+)
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    bugzilla,
+    skipif_ocs_version,
+)
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
```

Comment:
Since this test is related to pvcs, should be be Greensquad here?

Reply:
done